src/main/java/org/jenkinsci/test/acceptance/junit/AbstractJUnitTest.java

```diff
@@ -56,4 +69,9 @@ public int findAvailablePort() {
     protected void interrupt() {
         throw new RuntimeException("INTERACTIVE debugging");
     }
+
+    @After
+    public void injectSpec() {
+        supportBundle.addSpec(jenkins, SupportBundleRequest.builder().includeDefaultComponents().setOutputFile(diagnostics.touch("support-bundle.zip")).build());
```

Comment:
What happens if several tests are failing, could you have a filename collision ?

Reply:
No, the attachment is per test case

Reply:
the attachment is - but the junit attachment archiver only kicks in at the end running all the tests doesn't it?